pkg/fedvolumebackup/backup/backup_manager.go

```diff
@@ -425,14 +426,18 @@ func (bm *backupManager) teardownVolumeBackup(ctx context.Context, volumeBackup
 
 func (bm *backupManager) waitVolumeBackupComplete(ctx context.Context, volumeBackup *v1alpha1.VolumeBackup, backupMembers []*volumeBackupMember) error {
 	isBackupRunning := false
+	var failedBackups []*volumeBackupMember
 	for _, backupMember := range backupMembers {
-		if pingcapv1alpha1.IsVolumeBackupInitializeFailed(backupMember.backup) || pingcapv1alpha1.IsBackupFailed(backupMember.backup) {
+		if pingcapv1alpha1.IsVolumeBackupInitializeFailed(backupMember.backup) {
```

Comment:
are you sure only when all members have complete initialize phase can start real work on aws invocation?

Reply:
Yes, we execute initialization at first to pause GC and PD schedulers. After initialize successfully, we execute the actual backup phase. So if initialize failed, no volume snapshot will be created.